yarn.lock

```diff
@@ -251,9 +251,9 @@
     tslib "^2.0.0"
 
 "@azure/core-paging@^1.1.1":
-  version "1.4.0"
-  resolved "https://registry.yarnpkg.com/@azure/core-paging/-/core-paging-1.4.0.tgz#b04a73ad18149733a848c3089a5e5ed144592338"
-  integrity sha512-tabFtZTg8D9XqZKEfNUOGh63SuYeOxmvH4GDcOJN+R1bZWZ1FZskctgY9Pmuwzhn+0Xvq9rmimK9hsvtLkeBsw==
+  version "1.5.0"
```

Comment:
Can we revert changes to this file except for packages you need to update? Updating the entire yarn.lock to latest packages would require a lot of validation since you'd be updating all of our dependencies. You shouldn't need to regenerate the entire yarn.lock file if you're just updating one package.